typings/index.d.ts

```diff
@@ -1182,6 +1193,8 @@ declare module "noblox.js" {
      */
     function getGroup(groupId: number): Promise<Group>;
 
+    function getGroups(userId: number): Promise<IGroupPartial[]>
```

Comment:
Even though it's obvious, it would be helpful to add a description of what the function does in a comment above.